docker-compose.example.yml

```diff
@@ -4,18 +4,23 @@ services:
     image: postgres:11 # pin postgres to a major version
     environment:
       - PGDATA=/db-data
-      - POSTGRES_PASSWORD=postgres # has to correspond with POSTGRES_PASSWORD in APP
+      - POSTGRES_PASSWORD=postgres # has to correspond with DB_PASSWORD in APP
     volumes:
       - db:/db-data:rw
   app:
     image: babybuddy/babybuddy
     # See README.md#configuration for other environment configuration options.
     environment:
       - ALLOWED_HOSTS=
+      - DB_ENGINE=django.db.backends.postgresql
```

Comment:
Note that the default is actually `django.db.backends.postgresql_psycopg2`, not `django.db.backends.postgresql`. For the example file, lets stick to just providing the preset `DB_PASSWORD` variable.

Reply:
Whoops, thanks for catching that. Updating now. Also assuming you meant to remove the default variables as examples and leave it as 

DB_HOST=

but if not just let me know and I'll update.

Reply:
No, I mean lets just replace the existing parameter there and not add the others -- the current default works correctly without any additional setup and anyone who wants further configurability can refer to the docs.

Reply:
Ahhhh ok, now I'm following. Latest commit should roll back those changes, but let me know if I'm still missing the mark.